code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -27,7 +27,6 @@
 	. = ..()
 	if(air_contents.total_moles < 5)
 		user << "<span class='danger'>The meter on \the [src] indicates you are almost out of gas!</span>"
-		playsound(user, 'sound/effects/alert.ogg', 50, 1)

```

Comment:
A final alternative solution is to just play the sound to the person examining the object, instead of locally.